Example/Example Swift/CredentialsManager.swift

```diff
@@ -24,12 +24,12 @@ final class CredentialsManager {
             let client_email_domain = keys[clientEmailDomain] as? String,
             !client_id.isEmpty, !client_password.isEmpty, !client_email_domain.isEmpty {
             
-            return GiniClient(clientId: client_id,
-                              clientSecret: client_password,
-                              clientEmailDomain: client_email_domain)
+            return Client(id: client_id,
+                          secret: client_password,
+                          domain: client_email_domain)
         }
-        return GiniClient(clientId: "",
-                          clientSecret: "",
-                          clientEmailDomain: "")
+        return Client(id: "",
```

Comment:
I think I did the same in the past and Peter complained that I'm returning an empty Client and not a null or something. I tend to agree. A Client object with empty strings is not super useful.

Reply:
We actually discussed this already. It was an optional before and we changed it to this so the example app still works without credentials (avoiding bad first impressions).
Adding a warning there would help though.